package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "joi-image-extensions",
+  "version": "0.1.0",
```

Comment:
Looks solid enough to jump straight to v1.0.0 to me!

Reply:
After this gets merged I was going to just run an `npm version` to do just that.